### DISCUSSION TITLE
gminer source code

### DISCUSSION THREAD

**Discussion by Krokodiler23** [community/community#86115] [Programming Help]
### Select Topic Area

Question

### Body

hello everyone,

I'm looking for the gminer source code but can't find it, am I too stupid or is it not public? I only ever find the readme file!

thank you for your time
Manuel

**Comment by Shahnawazk7944** [comment]
**It's not a matter of intelligence at all!** The source code for Gminer, as of this moment, isn't publicly available. Let's explore the options and resources available to you:

**1. Official Channels:**

- **Check the official Gminer website and repository:** See if any recent updates or announcements mention a change in source code availability.
- **Contact the developers:** Reach out to them directly via their website or social media channels to inquire about source code access. They might have specific requirements or conditions for sharing it.

**2. Community Forums and Discussions:**

- **Explore community forums and discussion boards:** Engage with other Gminer users and developers to see if they have insights on source code availability or potential workarounds.
- **Search for discussions on GitHub or other code-sharing platforms:** You might find forks or unofficial versions of the code shared within the community.

**3. Alternative Approaches:**

- **Consider using a different mining software:** If having access to the source code is crucial for your purposes, explore other mining software with publicly available code. This way, you'll have full control over the software's behavior and customization.
- **Contribute to the project:** If you're interested in contributing to Gminer's development, reach out to the developers and express your willingness to help. This might open up opportunities to access or contribute to the code.

**Stay informed:** Keep an eye on the official Gminer channels for any updates or changes regarding source code availability.

**Reply by Akash1134** [reply]
Hi @Shahnawazk7944 , 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

**Comment by AidenJamesYt** [comment]
I believe that this might be what your looking for:
https://github.com/develsoftware/GMinerRelease
https://github.com/develsoftware/GMinerRelease/releases/tag/3.43

**Reply by Krokodiler23** [reply]
Thanks but unfortunately it's only the readme file where you can set the setup but not the source code!
